StripeFinancialConnections/StripeFinancialConnections/Source/Native/ManualEntry/ManualEntryFormView.swift

```diff
@@ -82,15 +87,24 @@ final class ManualEntryFormView: UIView {
         return (routingNumberTextField.text, accountNumberTextField.text)
     }
 
-    init() {
+    init(isTestMode: Bool) {
         super.init(frame: .zero)
         let contentVerticalStackView = UIStackView(
             arrangedSubviews: [
                 textFieldStackView,
             ]
         )
+
+        if isTestMode {
+            let testModeBanner = TestModeAutofillBannerView(
+                context: .account,
+                didTapAutofill: applyTestModeValues
+            )
+            contentVerticalStackView.insertArrangedSubview(testModeBanner, at: 0)
```

Comment:
_(this is completely subjective so feel free to ignore)_

one thing to consider is that now that we add multiple UIView's to the stack view, maybe it reads better like this (this is psuedocode):

```
var contentVerticalStackView = UIStackView()
if isTestMode {
   contentVerticalStackView.add(testModeBanner)
}
contentVerticalStackView.add(textFieldStackView)
```

the reason is just that one can read from top to bottom so it "flows better," while right now the `insertArrangedSubview(testModeBanner, at: 0)` is sort-of introducing a side-effect where the original UISTackView declaration gives the impression that `textFieldStackView` will be first:

```
let contentVerticalStackView = UIStackView(
    arrangedSubviews: [
         textFieldStackView,
    ]
)
```

...but the code later actually modifies this flow